packages/react/migrations.json

```diff
@@ -41,6 +41,12 @@
       "version": "18.1.1-beta.0",
       "description": "Ensure targetDefaults inputs for task hashing when '@nx/webpack:webpack' is used are correct for Module Federation.",
       "factory": "./src/migrations/update-18-1-1/fix-target-defaults-inputs"
+    },
+    "update-module-federation-ssr-server-file": {
+      "cli": "nx",
+      "version": "19-6-0-beta.4",
```

Comment:
`19.6.0-beta.4`

Reply:
Should be already there I think.

Reply:
No, `19-6-0-beta.4` is invalid, it needs to be dots.

Reply:
🤦🏿